src/OpenLoco/src/GameSaveCompare.cpp

```diff
@@ -8,6 +8,7 @@
 #include "OpenLoco.h"
 #include "S5/Limits.h"
 #include "S5/S5.h"
+#include "S5/S5Options.h"
```

Comment:
Guessing this is no longer needed, with files now being loaded into the internal `ScenarioOptions` instead?

Edit: Wait this is still comparing S5::GameState! Never mind then.